awx/ui_next/src/screens/Template/Survey/SurveyListItem.jsx

```diff
@@ -119,7 +119,11 @@ function SurveyListItem({
               )}
               {[question.type].includes('multiselect') &&
                 question.default.length > 0 && (
-                  <ChipGroup numChips={5}>
+                  <ChipGroup
+                    numChips={5}
+                    expandedText={i18n._(t`Show Less`)}
+                    collapsedText={i18n._(t`more`)}
```

Comment:
It seems that all the `ChipGroup` components will require this change. Otherwise, they will be using the default values.

Reply:
hey @dsesami, I realized that all `ChipGroup` usage should be updated to be coherent change. If you can take another look I appreciate.